### PR TITLE
[Feat] release notes for v3.0.1

### PR DESCRIPTION
**Pull request by MattDodsonEnglish**

### CODE REVIEW

File: content/releases/3-0-1.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+- Standardize config casing
+- Prohibit secrets and environment variables from having same ID. This fixes a bug in BPMN execution.
+- Fix `saveVersionAs` bug
+
+**Refactor**
+- Change config file from JSON to YAML
+- Remove unused properties
+- Use Rhize-Go service for Open Telemetry
+
+## Upgrade
```
[the remark this answers is not in the document]
Comment (MattDodsonEnglish) [reply content/releases/3-0-1.md:85]: or @tomhollingworth

File: content/releases/3-0-1.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+To upgrade to v3.0.1, first ensure that you have made corrections for the aforementioned breaking changes:
+
+Rhize now limits BPMN users to the `libreBpmn` audience and `bpmn:mutation` role.
+If users do not have this audience and role, they will unable to log in.
+To fix this:
+1. Log in to the Keycloak service in your Rhize environment.
+2. Select the `libre` realm.
+3. Configure the audience and role for the Users or groups that should be able to run BPMN workflows. For details, read [Install Keycloak]({{< relref "/deploy/install/keycloak/" >}}).
+
+Work calendars now query for `workCalendarDefinitions` not `workCalendars`.
```
Comment (MattDodsonEnglish) [inline content/releases/3-0-1.md:96]:
```suggestion
Work calendars now query for `workCalendarDefinitions`, not `workCalendars`.
```

File: content/releases/3-0-1.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+- Add 0-n relationship from `materialRequirements` to `materialActual`.
+- `segmentResponse` can now have multiple `jobResponses` attached.
+- `breaking` Change `stateModelTransition` from `state` to a relationship.
+- link `operationsEventDefinitionProperty` to `operationsEventProperty`.
+
+### Core
+
+**Features**
+- Add validation to config files
+- Obfuscate sensitive information
```
Comment (MattDodsonEnglish) [inline content/releases/3-0-1.md:71]:
```suggestion
- Obfuscate sensitive information when published
```

File: content/releases/3-0-1.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+- Reuse expected Audience for clients to match documented client name.
+- Do not create calendar entry for end events.
+- Allow `__traceDebug` to be set from [variable context]({{< relref "/how-to/bpmn/create-workflow/#access-process-variable-context" >}}).
+
+
+### Schema
+
+**Features**
+- `breaking`. Add `id` and `label` to `stateTransitionInstance`.
+- Add menu URL for portal in UI (preparation for future work in UI).
```
Comment (tomhollingworth) [inline content/releases/3-0-1.md:55]: Change to: Add `url` to Menu items to enable embedding low-code UIs in the Rhize Admin UI.

File: content/releases/3-0-1.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+- Add filter to query material actual by a material requirement and vice-versa.
+- Link `workCalendarEntries` to equipment versions.
```
Comment (tomhollingworth) [inline content/releases/3-0-1.md:57]: Add relationship between `equipmentVersion` and `workCalendarEntries`

File: content/releases/3-0-1.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+- Include Kafka in the enum values for data sources.
```
Comment (tomhollingworth) [inline content/releases/3-0-1.md:58]: Add Kafka to datasource enum

File: content/releases/3-0-1.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+- Added optional `InformationObjectTypeEnum` to `OperationsEventRecordEntry`.
+
+**Fixes**
+- Add 0-n relationship from `materialRequirements` to `materialActual`.
+- The `segmentResponse` entity can now have multiple `jobResponses` attached.
+- `Breaking`. Change `stateModelTransition` from `state` to a relationship.
+- link `operationsEventDefinitionProperty` to `operationsEventProperty`.
```
Comment (tomhollingworth) [inline content/releases/3-0-1.md:65]: Fix inverse relationship between `operationseventDefinitionProperty` and `operationsEventProperty`

File: content/releases/3-0-1.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+- `breaking`. Add `id` and `label` to `stateTransitionInstance`.
+- Add menu URL for portal in UI (preparation for future work in UI).
+- Add filter to query material actual by a material requirement and vice-versa.
+- Link `workCalendarEntries` to equipment versions.
+- Include Kafka in the enum values for data sources.
+- Added optional `InformationObjectTypeEnum` to `OperationsEventRecordEntry`.
+
+**Fixes**
+- Add 0-n relationship from `materialRequirements` to `materialActual`.
+- The `segmentResponse` entity can now have multiple `jobResponses` attached.
```
Comment (tomhollingworth) [inline content/releases/3-0-1.md:63]: Change SegementResponse relationship to JobResponse to be a 1:N

File: content/releases/3-0-1.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+- `Breaking`. Change `stateModelTransition` from `state` to a relationship.
+- link `operationsEventDefinitionProperty` to `operationsEventProperty`.
+
+### Core
+
+**Features**
```
Comment (tomhollingworth) [inline content/releases/3-0-1.md:69]: **BREAKING**: Change config file from JSON to YAML format

File: content/releases/3-0-1.md [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+- Create operations event on migrations.
+- Create `AppContext` to contain `context.Context`, `trace.Span`, and `logger` that can be passed around functions within the application.
+
+**Fixes**
+- Standardize config casing.
+- Prohibit secrets and environment variables from having the same ID. This fixes a bug in BPMN execution.
+- Fix `saveVersionAs` bug.
+
+**Refactor**
+- Change config file from JSON to YAML.
```
Comment (tomhollingworth) [inline content/releases/3-0-1.md:81]: The above line is a breaking change, throw it up the top, and add:

Change configuration keys to camel case and to consistent between microservices. (Backward compatible)

File: content/releases/3-0-1.md [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+- link `operationsEventDefinitionProperty` to `operationsEventProperty`.
+
+### Core
+
+**Features**
+- Add validation to config files.
+- Obfuscate sensitive information when published.
+- Create operations event on migrations.
+- Create `AppContext` to contain `context.Context`, `trace.Span`, and `logger` that can be passed around functions within the application.
+
```
Comment (tomhollingworth) [inline content/releases/3-0-1.md:74]: Add `getInformationObject` query resolver to GraphQL API

**Review by tomhollingworth — CHANGES_REQUESTED** [review]

**Pull request merged**